firefox-ios/Client/Frontend/Settings/Main/Debug/ToggleCloseRemoteTabs.swift

```diff
@@ -0,0 +1,28 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+class ToggleCloseRemoteTabs: HiddenSetting, FeatureFlaggable {
```

Comment:
Do we still need this Debug section? I think we're already in the new system they have in: Secret Menu -> Feature Flags -> Enable Close Remote Tabs

https://github.com/mozilla-mobile/firefox-ios/blob/7e5688231e532c568a3589625f6b4695ba160c91/firefox-ios/Client/Frontend/Settings/Main/Debug/FeatureFlags/FeatureFlagsDebugViewController.swift#L27